<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-B7UDF6D" ignored="0" name="Test" tests="5" failures="1" timestamp="2021-12-28T23:26:38 IST" time="0.225" errors="0">
  <testcase name="demo" time="0.025" classname="com.testng.features.Demo_Transform"/>
  <testcase name="demoone" time="0.006" classname="com.testng.features.Demo_Transform">
    <skipped/>
  </testcase> <!-- demoone -->
  <testcase name="demoone" time="0.009" classname="com.testng.features.Demo_Transform">
    <skipped/>
  </testcase> <!-- demoone -->
  <testcase name="demoone" time="0.007" classname="com.testng.features.Demo_Transform">
    <skipped/>
  </testcase> <!-- demoone -->
  <testcase name="demoone" time="0.006" classname="com.testng.features.Demo_Transform">
    <failure type="java.lang.AssertionError" message="expected [Karthick@1] but found [Karthick@123]">
      <![CDATA[java.lang.AssertionError: expected [Karthick@1] but found [Karthick@123]
at com.testng.features.Demo_Transform.demoone(Demo_Transform.java:20)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- demoone -->
</testsuite> <!-- Test -->
